Godeps/Godeps.json

```diff
@@ -896,6 +909,10 @@
 			"Rev": "c2528b2dd8352441850638a8bb678c2ad056fd3e"
 		},
 		{
+			"ImportPath": "golang.org/x/net/http2",
```

Comment:
What uses this?  In go 1.6 this is part of stdlib, was this a dependency we weren't taking before from grpc?  Is the bradfitz/http version no longer used?